sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/healthcare/FhirIO.java

```diff
@@ -220,6 +222,19 @@ public static Import importResources(
     return new Import(fhirStore, tempDir, deadLetterDir, contentStructure);
   }
 
+  /**
+   * Export resources to GCS. Intended for use on non-empty FHIR stores
+   *
+   * @param fhirStore the fhir store
```

Comment:
I think it's better to add a bit more details here, e.g. the format of an FHIR store name, and the GCS URI prefix etc.

Reply:
Done.